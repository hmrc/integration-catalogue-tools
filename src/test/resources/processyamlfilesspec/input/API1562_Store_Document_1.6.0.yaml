openapi: 3.0.3
info:
  title: Store Document
  description: 'This is a generic interface for storing binary documents on an enterprise document storage facility, currently Electronic Folders.'
  version: 1.6.0
  contact:
    name: Wayne Morris
    email: wayne.morris@hmrc.gov.uk
tags:
  - name: 'API#1562'
    description: API for storing a document.
