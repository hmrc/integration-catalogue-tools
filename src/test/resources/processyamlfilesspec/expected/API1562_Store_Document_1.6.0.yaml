openapi: 3.0.3
info:
  title: Store Document
  description: 'This is a generic interface for storing binary documents on an enterprise document storage facility, currently Electronic Folders.'
  version: 1.6.0
  contact:
    name: Wayne Morris
    email: wayne.morris@hmrc.gov.uk
  x-integration-catalogue:
    reviewed-date: 2022-04-22T20:27:05Z
    platform: CORE_IF
    publisher-reference: 1562
tags:
  - name: 'API#1562'
    description: API for storing a document.
